artifactory/commands/replication/create.go

```diff
@@ -68,7 +68,12 @@ func (rcc *ReplicationCreateCommand) Run() (err error) {
 		if key == "serverId" {
 			serverId = value.(string)
 		} else {
-			writersMap[key](&replicationConfigMap, key, value.(string))
+			for key, value := range replicationConfigMap {
```

Comment:
This loop iterates over the map, but it is inside an outer loop which also iterates the map.
Any reason why we should iterate the map n squared times?

Reply:
Remove duplicate for loop